js/dropdownAdapter.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'libraries/select2'
+], function() {
+	/*
+     * issue/1543: fix from https://github.com/select2/select2/issues/4063
+     */
+    var dropdownAdapter;
+    jQuery.fn.select2.amd.require([
+        "select2/utils",
+        "select2/dropdown",
+        "select2/dropdown/attachContainer",
+        "select2/dropdown/closeOnSelect"
+    ], function(Utils, DropdownAdapter, AttachContainer, CloseOnSelect) {
+
+        /*
+         * issues/1889: fix from https://github.com/adaptlearning/adapt_framework/issues/1889
+         * code added to make dropdown sit above body bottom where appropriate
+         */
+        AttachContainer.prototype.bind = function(decorated, container, $container) {
+
+            var self = this;
```

Comment:
Use instead: 
```js
function() {
}.bind(this)
```

Reply:
Note that this change will require a major version bump since it means this plugin will no longer be IE8 compatible (yay!)

Reply:
Actually I'm wrong about that... the ES5 shim we added in [Framework v2.0.16](https://github.com/adaptlearning/adapt_framework/releases/tag/v2.0.16) makes `Function.bind` available in IE8 :-(